app/controllers/admin/attachments_controller.rb

```diff
@@ -77,6 +84,10 @@ def attachable_attachments_path(attachable)
 
 private
 
+  def get_layout
+    preview_design_system_user? ? "design_system" : "admin"
```

Comment:
I think this needs to be scoped to the specific controller actions you're working on or it'll apply to all the controller actions (index etc.)

```

  def get_layout
    return "admin" unless preview_design_system_user?

    case action_name
    when "edit", "update", "new", "create"
      "design_system"
    else
      "admin"
    end
  end
```

Reply:
Ah right, makes sense 👍